src/common/security/utils/ram_policy.go

```diff
@@ -0,0 +1,55 @@
+package utils
```

Comment:
Should the ram related stuff be moved to `ram` package?

Also I'm thinking shall we move the `ram` into `security` package?

Reply:
`security.utils` pkg moved to `ram.project`